lib/resources/r4/appointment_patch.yaml

```diff
@@ -0,0 +1,21 @@
+---
+name: Appointment
+field_name_base_url: http://hl7.org/fhir/R4/appointment-definitions.html#Appointment
+operations:
+  - name: replace-status
+    path: /status
```

Comment:
I think the URLs here are a little messed up. The status link should take me to the Appointment status field, but it takes me to code

Reply:
fixed the link to status, https://github.com/cerner/fhir.cerner.com/pull/257/commits/8fe9222d7b73875cc586014df948cc587f7fcbc7#diff-07dbf4d3264a1e7a98f8d854b77f88e9R10